services/gitlab/gitlab-forks.tester.js

```diff
@@ -0,0 +1,20 @@
+import { isMetric } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+
+export const t = await createServiceTester()
+
+t.create('Forks')
+  .get('/gitlab-org/gitlab.json')
+  .expectBadge({
+    label: 'forks',
+    message: isMetric,
+    color: 'blue',
+    link: ['https://gitlab.com/gitlab-org/gitlab/-/forks/new'],
+  })
+
+t.create('Forks (project not found)')
+  .get('/user1/gitlab-does-not-have-this-repo.json')
+  .expectBadge({
+    label: 'forks',
+    message: 'project not found',
+  })
```

Comment:
Could we have a test for the private repo case? Also, we've overlooked it in the other reviews but none of these GitLab badges we have added recently include a test for the case where we pass a custom `baseUrl`. I think I've noticed it here because the links include `baseUrl`. We don't need to go back and deal with them all in this PR but it would be nice to have test cases for this so we don't accidentally break it in a future refactor. https://gitlab.gnome.org/ is a good example of a self-hosted GitLab server with lots of public projects. We've used it in some of the other gitlab test modules

Reply:
This was an oversight on my part, I added the `gitlab_url` parameter to the example, but did not provide a test case for the custom instance.